src/asciidoc/docs/customizations/media.adoc

```diff
@@ -0,0 +1,220 @@
+
+* *audio* - If you set `acceptOnly` to an array containing `audio`, the user will be prompted to choose a sound file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["audio"]` is equivalent to using a <<sound#, sound VCC>>.
+
+* *file* - If you set `acceptOnly` to an array containing `file`, the user will be prompted to choose a file for  the VCC.
+The file selected can be of any type.
+Using a `media` VCC with `acceptOnly` set to `["file"]` is equivalent to using a <<file#, file VCC>>.
+
+=== hideExtensions
+
+The `media` VCC type enables you to hide all asset packs and VCC extensions in cases where they don't make sense (for example, templates for selling premium assets).
+To use this feature in your templates, enable the `hideExtensions` type option.
+In the following code sample, extensions are hidden, so *Browse Image Asset Packs* will not appear as a selection.
+
+[source,json]
+----
+{
+    "key": "hideExtensions",
+    "name": "Image or Video No Extensions",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video"],
+        "hideExtensions": true,
+        "videoOptions": {
+            "hls": true
+        }
+    }
+}
+----
+
+The above sample code will display the following selections.
+
+image::mediaNoExtensionsVCC.png[title="Media No Extensions VCC"]
+
+=== videoOptions
+
+For delivery of longer content, HTTP Live Streaming (HLS) can be enabled by setting `hls` to true.
+When enabled, uploaded videos are transcoded for HLS and return an m3u8 playlist.
+Use this feature in conjunction with https://github.com/video-dev/hls.js/[hls.js] for controlling playback.
+
+Pose detection data can be generated by setting `estimatePoses` to true.
+When specified, pose data is available after upload by appending `.poses` to the returned URL.
+For example, `\https://objects.koji-cdn.com/project-id/my-video.mp4.poses`.
+
+=== returnType
+
+`returnType` takes a value of `url` or `extended`.
+If `returnType` is set to `url`, a string will be returned.
+If `returnType` is set to `extended`, an object will be returned.
+If `returnType` is not specified, it defaults to `extended`.
+
+==== Sample code for setting returnType to url
+
+[source,json]
+----
+{
+    "key": "urlExample"
+    "name": "Media Upload",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ['image'],
+        "returnType": "url",
+    }
+}
+----
+
+==== Sample code for retrieving and printing a URL returnValue 
+
+[source,javascript]
+----
+let url = this.instantRemixing.get(['settings', 'logo']);
+console.log("returnType: " + url);
+----
+
+==== Sample code for setting returnType to extended
+
+[source,json]
+----
+{
+    "key": "extendedExample"
+    "name": "Media Upload",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ['image', 'video', 'audio'],
+        "returnType": "extended",
+    }
+}
+----
+
+The object returned when `returnType` is set to `extended` has the following structure:
```

Comment:
This information is critical and it's getting lost in the other samples. When the rest of the samples are removed, it will stand out more.